script/playbackstrategy/nativestrategy.js

```diff
@@ -2,15 +2,15 @@ define('bigscreenplayer/playbackstrategy/nativestrategy',
   [
     'bigscreenplayer/playbackstrategy/legacyplayeradapter',
     'bigscreenplayer/models/windowtypes',
-    'bigscreenplayer/playbackstrategy/modifiers/html5',
+    'bigscreenplayer/playbackstrategy/modifiers/' + (window.bigscreenPlayer.mediaPlayer || 'html5'),
```

Comment:
Probably worth updating the documentation on the wiki about this new configurable option and maybe the difference in the media players.